global-variables.ps1

```diff
@@ -103,6 +115,8 @@ $GOLANG_URL_1_94 = "https://dl.google.com/go/go1.9.4.windows-amd64.msi"
 $DCOS_WINDOWS_GIT_URL = "https://github.com/dcos/dcos-windows.git"
 $MESOS_JENKINS_GIT_URL = "https://github.com/Microsoft/mesos-jenkins.git"
 $LIBSODIUM_GIT_URL = "https://github.com/jedisct1/libsodium.git"
+$WINDOWS_APACHEL_HTTP_SERVER_URL = "https://www.apachelounge.com/download/VC15/binaries/httpd-2.4.33-win64-VC15.zip"
+$WINDOWS_APACHEL_HTTP_SERVER_SHA256 = "A9F94DBA6AFFE3BD98FEC01EF77DC932C123E25E360D29D970CC2CDD9F5BA237"
```

Comment:
We will only pick up the updated one when the need arises.  We would need to redo this part.
However, it will be handled through pkgpanda for handling packaging update.

Reply:
I'm just concern when they bump the version and remove the link to the former version, but since this is handled via packaging it should be fine.

Thanks for the comment.